tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/api/rest/v1/controllers/TaskController.java

```diff
@@ -230,25 +230,7 @@ private void checkTaskImplementation(final LazySupplier<TaskDTO> taskSupplier) {
   }
 
   private boolean hasAccessToTask(final LazySupplier<TaskDTO> taskSupplier) {
-    final String userName = userReader.getCurrentUser().getUserId();
```

Comment:
TODO: revisit this, ideally we make use of the new authorizations here

Reply:
Needs to be reintroduced with https://github.com/camunda/product-hub/issues/2561